.github/workflows/tests.yaml

```diff
@@ -0,0 +1,66 @@
+name: Tests
+
+on:
+  push:
+    branches: ['*']
+    tags: ['v*']
+  pull_request:
+    branches: ['*']
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
```

Comment:
should this be trusty, just for the sake of parity?

Reply:
18.04 is the oldest supported so I figured just go latest
https://github.com/actions/virtual-environments